lib/src/cancelable_operation.dart

```diff
@@ -29,6 +29,10 @@ class CancelableOperation<T> {
   ///
   /// [onCancel] will be called synchronously when the operation is canceled.
   /// It's guaranteed to only be called once.
+  ///
+  /// This is the same as creating a [CancelableCompleter] and completing it
+  /// with [inner]. The [isCompleted] field will always be true, regardless of
+  /// whether the completer backing [inner] has completed.
```

Comment:
Generally, avoid using an unqualified "this" (or "it" or similar nondescript words) in documentation. It's almost always better to qualify.

Maybe:
```dart
/// Calling this constructor is equivalent to creating a [CancelableCompleter] 
/// and completing it with [inner]. As such, [isCompleted] is true from the moment
/// this [CancelableOperation] is created, regardless of whether [inner] has completed 
/// yet or not.

We don't know whether there is a completer backing `inner`, so we can't refer to it. It's just a future, they can come from anywhere.

Reply:
Done